packages/backend/src/health.ts

```diff
@@ -0,0 +1,28 @@
+import {HealthState} from "common";
+import fetch from "node-fetch";
+// Cannot be `import` as it's not under TS root dir
+const {version: VERSION} = require('../package.json');
+
+const filterByName = (result: { name: string }): boolean => {
+    return new RegExp('[0-9.]+$').test(result.name)
+}
+
+const sorterByDate = (a: { last_updated: string }, b: { last_updated: string }): number => {
```

Comment:
You can do this server side via the API but we don't have to trust the order

Reply:
The data is already sorted but I felt it was better to be sure.